src/components/v5/common/Table/Table.tsx

```diff
@@ -62,269 +101,288 @@ const Table = <T,>({
   const shouldShowEmptyContent = emptyContent && data.length === 0;
 
   return (
-    <div
-      className={clsx(
-        className,
-        'border border-gray-200 rounded-lg overflow-hidden',
+    <>
+      {title && (
```

Comment:
could we maybe just use the `TableHeader` component wherever we want the title? it's a fragment so it's not tied to any layout things, we could get rid of 3 more props here

Reply:
Hey man, what do you specifically mean by just using the TableHeader? We are already using it to render the title